.circleci/testcafe_search_bar.json

```diff
@@ -0,0 +1,5 @@
+{
+  "src": ["!tests/acceptance/acceptancesuites/*.js", "tests/acceptance/acceptancesuites/searchbaronlysuite.js"],
```

Comment:
can we remove `!tests/acceptance/acceptancesuites/*.js"`? I think just specifying tests/acceptance/acceptancesuites/searchbaronlysuite.js should ignore other test files

Reply:
updated!